lang/japanese.xml

```diff
@@ -1,214 +1,124 @@
-      <Text>労働者がプランテーションを建設できるようになる。[NEWLINE][NEWLINE][ICON_Gold] ゴールド+2。適切な資源がある場所にのみ建設可能。平地に建設された場合、[ICON_Production] 生産力+1。[NEWLINE][NEWLINE]高級資源のある場所に建設すると、都市はそれを利用できるようになる。</Text>
-    </Replace>
-    <Replace Tag="LOC_POLICY_INSULAE_DESCRIPTION" Language="ja_JP">
-      <Text>少なくとも1つの専門区域を持つ都市の [ICON_Housing] 住宅+1。</Text>
-    </Replace>
-    <Replace Tag="LOC_POLICY_LIBERALISM_DESCRIPTION" Language="ja_JP">
-      <Text>少なくとも1つの専門区域を持つ都市の [ICON_Amenities] 快適性+1。</Text>
-    </Replace>
-    <Replace Tag="LOC_BELIEF_PAGODA_EXPANSION2_DESCRIPTION" Language="ja_JP">
-      <Text>パゴダが建設可能になる ( [ICON_Faith] 信仰力+3、ターンごとの影響力ポイント+1)。</Text>
-    </Replace>
-    <Replace Tag="LOC_UNIT_INDIAN_VARU_DESCRIPTION" Language="ja_JP">
-      <Text>インド固有の古典時代の重騎兵ユニット。隣接する敵ユニットの [ICON_Strength] 戦闘力-5。視界+1。</Text>
-    </Replace>
-    
-    
-      
-    
-		<Replace Tag="LOC_TRAIT_CIVILIZATION_NOBEL_PRIZE_DESCRIPTION" Language="ja_JP">
-			<Text>偉人を獲得した際に[ICON_Favor]外交的支持50を得る。工場から[ICON_GreatEngineer]大技術者ポイント+1、大学から[ICON_GreatScientist]大科学者ポイント+1。また、図書館、大学、工房、工場と政府複合施設の建造物に対する[ICON_PRODUCTION]生産力+50%。スウェーデンがゲーム中に存在する場合、世界会議において3つの追加コンペティションが産業時代以降に登場するようになる。</Text>
+		<Row Tag="LOC_PROMOTION_COMBAT_FOREIGN_CONTINENT_DESCRIPTION" Language="ja_JP">
+			<Text>異大陸で攻撃する際にユニットの[ICON_Strength]戦闘力+5。</Text>
+		</Row>
+		<!-- = leader ability (Bull Moose)= -->
+		<Replace Tag="LOC_TRAIT_LEADER_ANTIQUES_AND_PARKS_DESCRIPTION" Language="ja_JP">
+			<Text>アピールが最高で自然遺産または山岳に隣接しているタイルから[ICON_SCIENCE]科学力+1。天文学を研究後、さらに[ICON_SCIENCE]科学力+1。アピールが最高で遺産または森林に隣接しているタイルから[ICON_CULTURE]文化力+1。啓蒙思想の発明後、さらに[ICON_CULTURE]文化力+1。国立公園のある都市は全タイルのアピール+1。中世の市の社会制度を発明すると労働者による植林が可能になる。</Text>
 		</Replace>
-		<Replace Tag="LOC_BELIEF_RELIGIOUS_IDOLS_DESCRIPTION" Language="ja_JP">
-			<Text>高級資源とボーナス資源を扱う鉱山による [ICON_Faith] 信仰力+3、[ICON_GOLD]ゴールド+3。</Text>
+		<!-- = leader ability (Rough Rider)= -->
+		<Replace Tag="LOC_TRAIT_LEADER_ROOSEVELT_COROLLARY_ROUGH_RIDER_DESCRIPTION" Language="ja_JP">
+			<Text>科学技術か社会制度が新たな時代に到達するたび全ユニットの [ICON_Strength] 戦闘力+1。(太古を含み、産業時代まで上がっていき、最大+5)。[ICON_TradeRoute]交易路を確立している都市国家に送った[ICON_Envoy]代表団が2つ分としてカウントされる。弾道学の研究を完了すると固有ユニットであるラフライダーが解禁される。</Text>
+		</Replace>
+    <Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_ANCIENT" Language="ja_JP">
+      <Text>太古時代に到達していることにより [ICON_Strength] 戦闘力+1(ルーズベルトの効果)</Text>
+    </Row>
+    <Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_CLASSICAL" Language="ja_JP">
+      <Text>古典時代に到達していることにより [ICON_Strength] 戦闘力+2(ルーズベルトの効果)</Text>
+    </Row>
+    <Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_MEDIEVAL" Language="ja_JP">
+      <Text>中世時代に到達していることにより [ICON_Strength] 戦闘力+3(ルーズベルトの効果)</Text>
+    </Row>
+    <Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_RENAISSANCE" Language="ja_JP">
+      <Text>ルネサンス時代に到達していることにより [ICON_Strength] 戦闘力+4(ルーズベルトの効果)</Text>
+    </Row>
+    <Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_INDUSTRIAL" Language="ja_JP">
+      <Text>産業時代に到達していることにより [ICON_Strength] 戦闘力+5(ルーズベルトの効果)</Text>
+    </Row>
```

Comment:
Please check indents from the start of line.
It doesn't break file and not necessary now, but it doesn't according to file style and looks strange for others.